src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2938,19 +2948,19 @@ because xbuild doesn't support framework reference assemblies.
 	<_PrepareBuildApkDependsOnTargets>
 		_SetLatestTargetFrameworkVersion;
 		_GetLibraryImports;
-		_RemoveRegisterAttribute;
 		_CopyIntermediateAssemblies;
 		_CopyPdbFiles;
 		_CopyMdbFiles;
 		_LinkAssemblies;
+		_PrepareAssemblies;
```

Comment:
Should this be running after linkassemblies? The other chunk has it before. Just checking...

Reply:
I have a feeling @jonathanpeppers and I need to have a review next week and see what depends on what. `_PrepareAssemblies` is looking at the directory which `_LinkAssemblies` creates, so how it would work running before it I have no idea 🤷‍♀

Reply:
This reordering here seems like it isn't needed? Do we need to change `$(_PrepareBuildApkDependsOnTargets)` at all here?

It might explain some of the ordering issues @mattleibow saw?